fixtures/corgi_playwright.py

```diff
@@ -0,0 +1,39 @@
+from playwright.sync_api import sync_playwright
+import pytest
+
+
+@pytest.fixture
+def chrome_browser():
+
+    playwright_sync = sync_playwright().start()
+    chromebrowser = playwright_sync.chromium.launch(headless=False, slow_mo=1000)
+
+    return chromebrowser
+
+
+@pytest.fixture
+def chrome_page(chrome_browser, corgi_user, corgi_password):
+    context = chrome_browser.new_context(
+        http_credentials={"username": corgi_user, "password": corgi_password}
+    )
+    page = context.new_page()
+
+    return page
+
+
+@pytest.fixture
+def corgi_user(request):
```

Comment:
We can have a method 
```
def attribute_from_request(request, attribute):
    config_option = request.config.getoption(attribute)
    return config_option if config_option else request.config.getini(attribute)
  
@pytest.fixture
def corgi_user(request):
     return attribute_from_request(request, "corgi_user")
     
@pytest.fixture
def corgi_password(request):
     return attribute_from_request(request, "corgi_password")

Reply:
I will look into this but for now I leave it as is, if you don't mind

Reply:
Not a problem.